src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -463,24 +464,24 @@ const WorkflowView = _.flow(
         div({ style: { whiteSpace: 'pre' } }, ['  |  Drag or click to ']),
         linkButton({ onClick: () => this.uploader.current.open() }, ['upload json'])
       ]),
-      div({ style: { flex: '1 0 500px' } }, [
-        h(WorkflowIOTable, {
-          which: key,
-          includeOptionalInputs,
-          inputsOutputs,
-          config: modifiedConfig,
-          errors,
-          onChange: canCompute ? ((name, v) => this.setState(_.set(['modifiedConfig', key, name], v))) : undefined,
-          onSetDefaults: canCompute && key === 'outputs' ? () => {
-            this.setState(_.update(['modifiedConfig', 'outputs'], _.flow(
-              _.toPairs,
-              _.map(([k, v]) => [k, v || `this.${_.last(k.split('.'))}`]),
-              _.fromPairs
-            )))
-          } : undefined,
-          suggestions
-        })
-      ])
+      filteredData.length !== 0 ?
```

Comment:
If I understand the pattern correctly, you can do `filteredData.length !== 0 && div(...)` so you don't need the `: undefined`.

Reply:
Cool! That works, thanks!